spec/features/users/sign_in_spec.rb

```diff
@@ -99,10 +99,6 @@
     scenario 'user sees warning before session times out' do
       expect(page).to have_css('#session-timeout-msg')
 
-      request_headers = page.driver.network_traffic.flat_map(&:headers).uniq
-      ajax_headers = { 'name' => 'X-Requested-With', 'value' => 'XMLHttpRequest' }
-
-      expect(request_headers).to include ajax_headers
```

Comment:
I don't think this is possible to test with Chrome webdriver, but I don't think it's necessary to test.